charts/shoot-core/components/charts/monitoring/charts/node-exporter/templates/node-exporter.yaml

```diff
@@ -29,6 +29,8 @@ metadata:
   name: node-exporter
   namespace: kube-system
   labels:
+    networking.gardener.cloud/to-public-networks: allowed
+    networking.gardener.cloud/from-seed: allowed
```

Comment:
They are not needed in `Demonset.metadata.labels` they must only be in `Demonset.spec.template.metadata.labels`. Remove them here.

Reply:
done with both changes, thanks